lib/web_ui/dev/screenshot_manager.dart

```diff
@@ -0,0 +1,148 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// @dart = 2.6
+import 'dart:io' as io;
+import 'dart:convert';
+import 'dart:math';
+
+import 'package:image/image.dart';
+import 'package:webkit_inspection_protocol/webkit_inspection_protocol.dart'
+    as wip;
+import 'package:yaml/yaml.dart';
+
+import 'common.dart';
+
+/// [DriverManager] implementation for Chrome.
+///
+/// This manager can be used for both macOS and Linux.
```

Comment:
I never tested on Windows. Since I don't have bandwidth now to add the other missing parts of screenshot testing to windows it's out of scope of this work.

Reply:
Ah, makes sense. Maybe format it as a "TODO: make this work on Windows"?

Reply:
done.